apps/opik-backend/src/main/resources/liquibase/db-app-state/migrations/000032_add_dataset_versions_tables.sql

```diff
@@ -0,0 +1,52 @@
+--liquibase formatted sql
+--changeset idoberko2:add_dataset_versions_tables
+--comment: Create dataset_versions and dataset_version_tags tables for Git-like versioning
+
+CREATE TABLE IF NOT EXISTS dataset_versions (
+    id CHAR(36) NOT NULL,
+    dataset_id CHAR(36) NOT NULL,
+    version_hash VARCHAR(64) NOT NULL,
+    items_count INT DEFAULT 0,
+    items_added INT DEFAULT 0,
+    items_modified INT DEFAULT 0,
+    items_deleted INT DEFAULT 0,
+    change_description TEXT,
+    metadata JSON,
+    created_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6),
+    created_by VARCHAR(100) NOT NULL DEFAULT 'admin',
+    last_updated_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6),
+    last_updated_by VARCHAR(100) NOT NULL DEFAULT 'admin',
+    workspace_id VARCHAR(150) NOT NULL,
+    CONSTRAINT `dataset_versions_pk` PRIMARY KEY (id),
+    CONSTRAINT `dataset_versions_dataset_hash_uk` UNIQUE (dataset_id, version_hash),
```

Comment:
Add workspace_id

Reply:
Done in https://github.com/comet-ml/opik/pull/4019/commits/69eb2525d7b89628b47aec861b6bb8ee8b997cee